app/pages/collections/list.cjsx

```diff
@@ -65,6 +65,9 @@ List = React.createClass
       .then (collections) =>
         @setState {collections}
 
+  shared: (collection) ->
```

Comment:
Spoke with @zwolf and he clarified the functionality here for me, so I understand the suggestion now. Comparing to user id is what should be done.